apps/devhub/forms.py

```diff
@@ -30,7 +30,8 @@
 from translations.fields import TransTextarea, TransField
 from translations.models import delete_translation, Translation
 from translations.forms import TranslationFormMixin
-from versions.models import License, Version, ApplicationsVersions
+from versions.models import (License, Version, ApplicationsVersions,
+                             VALID_SOURCE_EXTENSIONS)
```

Comment:
I know it wasn't in the correct order previously, but please reorder the imports alphabetically while you're at it